src/transformers/models/segformer/modeling_segformer.py

```diff
@@ -124,7 +124,7 @@ def extra_repr(self) -> str:
 class SegformerOverlapPatchEmbeddings(nn.Module):
     """Construct the overlapping patch embeddings."""
 
-    def __init__(self, patch_size, stride, num_channels, hidden_size):
+    def __init__(self, config, patch_size, stride, num_channels, hidden_size):
```

Comment:
Need this new argument so we can use eps=config.layer_norm_eps. As this is an internal class, should be fine.